src/structure/pure/_common.js

```diff
@@ -401,7 +401,7 @@ function VisuDesign() {
     if( $element.attr('flavour') ) flavour = $element.attr('flavour');// sub design choice
     if( flavour ) classes += ' flavour_' + flavour;
     if($element.attr('class')) classes += ' custom_' + $element.attr('class');
-    var label = this.extractLabel( $element.find('label')[0], flavour );
+    var label = (widgetType==='text')?this.extractLabel( $element.find('label')[0], flavour, '' ):this.extractLabel( $element.find('label')[0], flavour );
```

Comment:
I'm not too happy about special casing a generic function (anything in _common.js is generic). A better solution is usually a generalisation.

But I'll pull anyway, we can fix that later.